data/assets/examples/renderable/renderablenodearrow/renderablenodearrow.asset

```diff
@@ -0,0 +1,41 @@
+-- Base Example
+-- This example shows an arrow pointing from one scene graph node in the direction of
+-- another. Here, it points from Earth to Mars.
+--
+-- Note that the arrows are generated as objects in 3D space and need to have a size
+-- that is suitable for the scene graph nodes they refer to. Here it is set based on
+-- the size of the Earth.
+
+local earth = asset.require("scene/solarsystem/planets/earth/earth")
+local mars = asset.require("scene/solarsystem/planets/mars/mars")
+
+local EarthRadius = 6371000.0
```

Comment:
How about moving those values into the Offset and Length value instead, making them:
` 2 * 6371000` and `5 * 6371000`?   One less variable simplifies the example a bit and we are already using a magic number for the `Width`